package.json

```diff
@@ -9,10 +9,10 @@
     "package-prebuilt": "node-pre-gyp package",
     "publish-prebuilt": "node-pre-gyp-github publish",
     "publish-all-prebuilt": "node scripts/publish-all-prebuilt.js",
-    "install": "node ./get-library.js && ( node-pre-gyp install --fallback-to-build=false || node build.js ) && node ./check-jlink-arm.js",
+    "install": "node scripts/pre-install.js && ( node-pre-gyp install --fallback-to-build=false || node build.js ) && node scripts/post-install.js",
```

Comment:
Have we considered splitting this into `preinstall`, `install` and `postinstall`? (as per https://docs.npmjs.com/misc/scripts ) Would that work?

Reply:
That is a good point. This is what I wanted to do originally, but unfortunately it did not work that well. The `preinstall` script is run before dependencies are installed, so pre-install.js will not be able to require f.ex. tar and sander.